app/helpers/templateGenerator.js

```diff
@@ -202,7 +202,7 @@ const generateTemplate = ({
       <h1>${componentName} ${templateType}</h1>
 
       <div>
-        <form method="post" action="/${templateType}/${componentType ? `${componentType}/components/` : ''}${name}" id="try-params" class="nhsuk-u-clear">
+        <form method="post" action="/${templateType}s/${componentType ? `${componentType}/` : ''}${name}" id="try-params" class="nhsuk-u-clear">
```

Comment:
yikes what that sneaky little `s` about?

Reply:
🤷 works now though

Reply:
do all the other templates and components still work. I don't like that sneaky `s` do we need to change the name of something so we don't have to do this

Reply:
this is the template generator - it will work for all now - it was not working at all before